core/drivers/firewall/stm32_rifsc.c

```diff
@@ -286,6 +303,41 @@ static TEE_Result stm32_rifsc_dt_conf_rimu(const void *fdt, int node,
 	return TEE_SUCCESS;
 }
 
+#if defined(CFG_STM32MP25)
+static TEE_Result stm32_rifsc_dt_conf_risal(const void *fdt, int node,
+					    struct rifsc_platdata *pdata)
+{
+	const fdt32_t *cuint = NULL;
+	int i = 0;
+	int len = 0;
+
+	cuint = fdt_getprop(fdt, node, "st,risal", &len);
+	if (!cuint) {
+		DMSG("No RISAL configuration in DT");
+		pdata->nrisal = 0;
+		return TEE_ERROR_ITEM_NOT_FOUND;
```

Comment:
Would it make sense to return `TEE_SUCCESS` here?

Reply:
I prefer to return not found and handle this error code afterwards to be coherent on whether the conf is present or not, is that ok?

Reply:
That's ok but managing `pdata->nrisal == 0` later seems sufficient (at least to me).